### DISCUSSION TITLE
RaMiX v1

### DISCUSSION THREAD

**Discussion by twofaktor** [minibolt-guide/ramix-node#3] [Announcements]
# RaMiX v1 has been released! 🍓🚀

![image](https://github.com/user-attachments/assets/337a6b70-e041-4944-a632-4869f7a7f620)

The RaMiX guide is available at ~~ > [ramix.minibolt.info](https://ramix.minibolt.info) < ~~

---

Based on [MiniBolt guide](https://github.com/minibolt-guide/minibolt) 

>_A step-by-step & updated guide to building a Bitcoin & Lightning node, and other stuff on a Raspberry Pi_

## Main changes to [RaspiBolt v3](https://github.com/raspibolt/raspibolt)

This guide has been adapted from RaspiBolt v3, those are the most relevant changes:

- Changed OS from **Raspberry Pi OS Lite Bullseye** (Debian 11 based - 64-bits) to Raspberry Pi OS Lite **Bookworm** (Debian 12 based - 64-bits).
- **Adapted to Raspberry Pi 5**, with the possibility of using internal storage: a PCIe to M.2 adapter + SSD NVMe.
- **Updated all services** that have been tested until now, to the latest version.
- Changed I2P, Fulcrum, and ThunderHub guides, to be **part of the core guide**.
- Included all UI & UX improvements of the MiniBolt guide.
- Fixed some links and wrong command issues.
- Some existing guides have been improved to **clarify the flow of steps to follow**.

## Other notes
-> The RRSS will be the same as the MiniBolt original project. Check -> [HERE](https://ramix.minibolt.info/#community)
-> The common resources like the Roadmap or Networkmap have been merged and will be used together. Check -> [HERE](https://ramix.minibolt.info/#resources)
-> The attempt to upgrade from Bullseye to Bookworm (RaspiBolt to RaMiX migration) has failed due to **several difficult-to-resolve dependency conflicts**, so unfortunately, there will be no dedicated migration guide and **only the possibility to start from scratch** ☹️

⚠️ Attention‼️-> This guide is in the WIP (work in progress) state and hasn't been completely tested yet. Many steps may be incorrect. Pay special attention to the "**Status: Not tested on RaMiX**" tag at the beginning of the guides. Be careful and act behind your responsibility.

For Raspberry Pi lovers!❤️🍓

<hr /><em>This discussion was created from the release <a href='https://github.com/minibolt-guide/ramix-node/releases/tag/v1.0'>RaMiX v1</a>.</em>

**Comment by ghost** [comment]
Thanks for the initiative loving it so far.
